conf/agent.routes

```diff
@@ -18,16 +18,6 @@ GET         /client-relationship-problem        controllers.agent.ClientRelation
 GET         /utr-problem                        controllers.agent.UTRErrorController.show
 POST        /utr-problem                        controllers.agent.UTRErrorController.submit
 
-GET         /language/:lang                     controllers.agent.AgentLanguageController.switchToLanguage(lang: String)
-
-# Enter Clients UTR
-GET         /client-utr                         controllers.agent.EnterClientsUTRController.show
-POST        /client-utr                         controllers.agent.EnterClientsUTRController.submit
-
-# Confirm Client
-GET         /confirm-client                     controllers.agent.ConfirmClientUTRController.show
-POST        /confirm-client                     controllers.agent.ConfirmClientUTRController.submit
-
```

Comment:
Why these routes are deleted?

Reply:
They were duplicated routes, if you see the rest of the file, you will see that multiple of the same route exist. Likely from a previous merge conflict which was resolved incorrectly.